addon/components/o-s-s/alert.hbs

```diff
@@ -11,6 +11,9 @@
       {{#if @subtitle}}
         <span class="subtitle">{{@subtitle}}</span>
       {{/if}}
+      {{#if (has-block "extra-content")}}
```

Comment:
The "main-container" class is defined as fx-row ; you might want to add an fx-col wrapping everything so that we don't get bad surprises when using the extra content !

Reply:
@Miexil it's in the `text-container` here. which is defined as `col`

Reply:
Do you want I move the block in `main-container` and did the update or keep it in `text-container` @Miexil ?